packages/fast-development-site-react/src/components/site/dev-tools.tsx

```diff
@@ -0,0 +1,310 @@
+import * as React from "react";
+import { toPx } from "@microsoft/fast-jss-utilities";
+import manageJss, { ComponentStyles, IJSSManagerProps, IManagedClasses } from "@microsoft/fast-jss-manager-react";
+import { IDevSiteDesignSystem } from "../design-system";
+
+export type frameworkType = "react" | "angular";
```

Comment:
Enums would probably work better - then you can actually use the value of the enum.